README.md

```diff
@@ -10,32 +10,36 @@ You can test React(ReactNative) runtime performance by using this lib. If you wa
 
 ## Table of Contents
 
-- [The problem](#the-problem)
-- [The solution](#the-solution)
-- [Installation](#installation)
-- [Example](#example)
-  - [count renders](#count-renders)
-  - [measure render time](#measure-render-time)
-- [API](#api)
-  - [perf](#perf)
-    - [renderCount](#renderCount)
-    - [renderTime](#renderTime)
-  - [cleanup](#cleanup)
-  - [ReactNative](#reactnative)
-  - [TypeScript](#typescript)
-- [Tips](#tips)
-  - [Performance](#performance)
-  - [Anonymous Component](#anonymous-component)
-  - [Hooks](#hooks)
-- [LICENSE](#license)
+- [react-performance-testing](#react-performance-testing)
+  - [Table of Contents](#table-of-contents)
```

Comment:
I think these items don't need to include on the table of contents.

Reply:
done